src/test/script_tests.cpp

```diff
@@ -180,6 +180,10 @@ void DoTest(const CScript& scriptPubKey, const CScript& scriptSig, const CScript
     }
 
 #if defined(HAVE_CONSENSUS_LIB)
+    // Make sure bitcoinconsensus_version() is referenced at least once from
```

Comment:
The comment does not convey the "root why", which I guess is "to avoid the risk of it becoming flagged as unused" or something along those lines. (Maybe overkill for most people, but why not be as clear as possible... )

Reply:
Thanks for reviewing! Fixed!